guide/plg_atTheTelescope.tex

```diff
@@ -921,7 +921,7 @@ \subsection{INDI}
 \item IOptron GotoNova Upgrade Kit 8400
 \end{itemize}
 
-\subsection{ASCOM (Windows only)}
+\subsection{ASCOM (Windows and Qt5-based builds only)}
```

Comment:
Without "and" it will be less ambiguous.

Reply:
It's a simple Boolean "and". I can live with a comma instead.